build.gradle.kts

```diff
@@ -17,6 +17,15 @@ import org.json.JSONObject
 // aims to make it easier to test publisher changes without having to override the artifacts.
 val publisherVersion = "v5"
 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.json:json:20180813")
+    }
+}
```

Comment:
AGP in version `7.x` provided `org.json` dependency (transitively):
![Screenshot 2024-02-02 at 12 05 31](https://github.com/wordpress-mobile/react-native-libraries-publisher/assets/5845095/9c46810c-a902-4950-8b3e-87fe1a21dae8)

With AGP `8.2.2`, `org.json` is no longer available. I've added this dependency in the same version for now, but probably a better solution would be to use built-in JSON parser: `groovy.json`.